index.html

```diff
@@ -0,0 +1,318 @@
+            <div id="form-text">
+                <p id="form-text-item">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <!-- form -->
+            <form action="https://formspree.io/f/myyawpqy" method="POST" id="form-content">
+                <div id="first-form-row">
+                    <input type="text" required name="first-name" placeholder="First Name" class="form-row-column">
+                    <input type="text" required name="last-name" placeholder="Last Name" class="form-row-column">
+                </div>
+                <input type="text" required name="full name" placeholder="Full Name" class="form-row" id="ignoreindesktop">
+                <input type="email" required name="email" placeholder="Email Address" id="" class="form-row">
+                <textarea name="message" id="" cols="30" rows="30" placeholder="Typing..." class="textareas"></textarea>
+                <div class="button-div">
+                    <button type="submit" class="form-submit-button"><p class="form-submit-button-text">Get in touch </p></button>
+                </div>
+            </form>       
+            <img src="img/desktop-backgrounds/bottom-left.svg" class="desktop-bottom-left-background mobile-dont" alt="">
+            <img src="img/desktop-backgrounds/bottom-leftmost.svg" class="desktop-bottom-leftmost-background mobile-dont" alt="">
+            <img src="img/desktop-backgrounds/bottom-right.svg" class="desktop-bottom-right-background mobile-dont" alt=""> 
+        </section>
+        
+        <!-- bottom-navigation -->
+        <section id="bottom-navigation">
+            <!-- line -->
+            <div class="bottom-horizontal-line"></div>
+            <!-- social media -->
+            <div class="social-media">
+                <section class="bottom-social-media-item"><a href="http//:github.com/francis450" target="_blank"><img src="img/social-media-icons/github.svg"></img></a></section>
+                <section class="bottom-social-media-item"><a href="http//:linkedin/in/njoroge-francis" target="_blank"><img src="img/social-media-icons/linkedin.svg"></img></a></section>
+                <section class="bottom-social-media-item"><a href="http//:evangelist" target="_blank"><img src="img/social-media-icons/angelist.svg"></img></a></section>
+                <section class="bottom-social-media-item"><a href="http//:twitter.com/nkamandefrancis" target="_blank"><img src="img/social-media-icons/twitter.svg"></img></a></section>
+                <section class="bottom-social-media-item"><a href="http//:medium.com" target="_blank"><img src="img/social-media-icons/medium.svg"></img></a></section>
+            </div>
+            <div class="bottom-email">
+                <a class="myemail" href="mailto:franciskamande2001@gmail.com">
+                    <b>franciskamande2001@gmail.com</b>
+                </a>
+            </div>
+        </section>
```

Comment:
- [ ] [OPTIONAL] Please decrease the margin from the bottom navigation to make it look like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=LyjfpMcIC7G9nUm4-0)

Figma
<img width="847" alt="Screenshot 2023-02-22 at 11 25 05 AM" src="https://user-images.githubusercontent.com/44798044/220543725-7cfd1946-56fa-4e61-b820-94af90d0a886.png">

Yours

<img width="1468" alt="Screenshot 2023-02-22 at 11 25 20 AM" src="https://user-images.githubusercontent.com/44798044/220543775-79a9c99b-2f78-4b65-a5ad-0e9539282755.png">